Basket-Donation-Payments/Basket-Donation-Payments/Models/FeaturedValueWeight.m

```diff
@@ -0,0 +1,27 @@
+//
+//  FeaturedValueWeight.m
+//  Basket Donation Payments
+//
+//  Created by Kristie Huang on 8/3/20.
+//  Copyright © 2020 Kristie Huang. All rights reserved.
+//
+
+#import "FeaturedValueWeight.h"
+
+@implementation FeaturedValueWeight
+
+@dynamic numberOfDonationsWeight;
+@dynamic predeterminedEventRelevancyWeight;
+@dynamic userFavoritesWeight;
+
++ (FeaturedValueWeight*)initNewFeaturedValueWeights {
+    FeaturedValueWeight *fvw = [FeaturedValueWeight new];
```

Comment:
You can delete this method and just call `[FeaturedValueWeight new]` when you want to create a new instance.